StRoot/StBFChain/BigFullChain.h

```diff
@@ -1587,6 +1587,7 @@ Bfc_st BFC[] = { // standard chains
                                                                                                     kFALSE},
   {"fstHit",     "", "fstChain", "event,fstCluster", "StFstHitMaker",       "StFstHitMaker","FST Hit Maker",
                                                                                                     kFALSE},
+  {"fstMuRawHit", "", "fstChain", "CMuDST",                     "","","Store FST Raw Hit in MuDst", kFALSE},
```

Comment:
Hm....this probably doesn't need to be included in `fstChain` because it implies that the makers invoked by `CMuDST` would be part of the `fstChain`. I haven't tested this, but I'm guessing you can have an empty string for the third element.

Reply:
Done.